include/mbedtls/ssl.h

```diff
@@ -1471,7 +1490,7 @@ struct mbedtls_ssl_context
 #endif /* MBEDTLS_SSL_DTLS_HELLO_VERIFY && MBEDTLS_SSL_SRV_C */
 
     /*
-     * Secure renegotiation
+     * Secure renegotiation (continued)
```

Comment:
I don't think the "continued" is correct here.

Reply:
Done.